client/my-sites/checkout/composite-checkout/components/concierge-refund-policy.jsx

```diff
@@ -23,7 +23,7 @@ class ConciergeRefundPolicy extends React.Component {
 
 	renderPolicy = () => {
 		const message = this.props.translate(
-			'You understand that {{refundsSupportPage}}Quick Start Session refunds{{/refundsSupportPage}} are limited to 30 days from the date of purchase.',
+			'You understand that {{refundsSupportPage}}Quick Start Session refunds{{/refundsSupportPage}} are limited to 14 days from the date of purchase.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.